includes/manage_book.inc

```diff
@@ -64,28 +64,29 @@ function islandora_book_ingest_zipped_pages(AbstractObject $object) {
  *   Drupal form definition.
  */
 function islandora_book_zipped_upload_form(array $form, array &$form_state, $book_pid) {
-  module_load_include('inc', 'islandora_ocr', 'includes/utilities');
   module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
   $book_object = islandora_object_load($book_pid);
   $current_pages = islandora_paged_content_get_pages($book_object);
   $last_page_number = count($current_pages);
-  $languages = module_exists('islandora_ocr') ? islandora_ocr_get_enabled_tesseract_languages() : array();
   $upload_size = min((int) ini_get('post_max_size'), (int) ini_get('upload_max_filesize'));
   $extensions = array('zip');
   $form = array();
   $derivatives = variable_get('islandora_book_ingest_derivatives', array('ocr'));
-  $do_ocr = in_array('ocr', $derivatives);
   $message = t("This sequence currently has @count pages. Additional pages will be appended to the end of the sequence by default. !break", array("@count" => $last_page_number, '!break' => '<br />'));
   $message .= t("Choose a number lower than @count to insert page(s) at a specific location in the sequence.", array("@count" => $last_page_number, '!break' => '<br />'));
 
-  $form['language'] = array(
-    '#access' => module_exists('islandora_ocr') && $do_ocr,
-    '#title' => t('Language'),
-    '#type' => 'select',
-    '#description' => t('Please select the language the page is written in.'),
-    '#options' => $languages,
-    '#default_value' => 'English',
-  );
+  if (module_exists('islandora_ocr') && in_array('ocr', $derivatives)) {
+    module_load_include('inc', 'islandora_ocr', 'includes/utilities');
+    $form['language'] = array(
+      '#access' => module_exists('islandora_ocr') && $do_ocr,
```

Comment:
Looks like you removed the setting of `$do_ocr` above. Should it be removed here. In fact based on the if statement determining if the form element is added, perhaps you could do `'#access' => TRUE,`?

Reply:
Whoa, sorry I missed this :/ yeah, I think I'm just gonna remove this line